atsign-foundation/at_server
# feat: Sync skip deletes until

**- What I did**
- added ability to skip delete commit entries in sync response. If skipDeletes flag is set to true, delete commits will not be returned in sync response from server to client.
**- How I did it**
- introduced skipDeletes flag in sync:from syntax
- added optional flag skipDeletes=false in commitLog getEntries. If set to true, delete commits will be skipped. if not passed, then existing behaviour applies. Refactored the code to introduce a strategy pattern to fetch keys.
- added unit tests
**- How to verify it**
- tests should pass

persistence change in this PR:
https://github.com/atsign-foundation/at_server/pull/2166
